test/unit/org/open.test.ts

```diff
@@ -193,36 +194,6 @@ describe('org:open', () => {
       expect(spies.get('requestGet').callCount).to.equal(1);
       expect(spies.get('requestGet').args[0][0]).to.deep.equal(`${testOrg.instanceUrl}/services/oauth2/singleaccess`);
     });
-
-    it('handles api error', async () => {
-      $$.SANDBOX.restore();
-      const mockError = new Error('Invalid_Scope');
-      mockError.name = 'Invalid_Scope';
-      $$.SANDBOX.stub(Connection.prototype, 'requestGet').throws(mockError);
-      try {
-        await OrgOpenCommand.run(['--targetusername', testOrg.username]);
-        expect.fail('should have thrown Invalid_Scope');
-      } catch (e) {
-        assert(e instanceof SfError, 'should be an SfError');
-        expect(e.name).to.equal('Invalid_Scope');
-        expect(e.message).to.equal(sharedMessages.getMessage('SingleAccessFrontdoorError'));
-      }
-    });
-
-    it('handles invalid responde from api', async () => {
-      $$.SANDBOX.restore();
-      $$.SANDBOX.stub(Connection.prototype, 'requestGet').resolves({
-        invalid: 'some invalid response',
-      });
-      try {
-        await OrgOpenCommand.run(['--targetusername', testOrg.username]);
-        expect.fail('should have thrown Invalid_Scope');
-      } catch (e) {
-        assert(e instanceof SfError, 'should be an SfError');
-        expect(e.message).to.equal(sharedMessages.getMessage('SingleAccessFrontdoorError'));
-        expect(e.data).to.contain({ invalid: 'some invalid response' });
-      }
-    });
```

Comment:
I don't know what these tests were supposed to do, the removed method with `SingleAccessFrontdoorError` was just a wrapper so 🤷🏼